client/src/index.jsx

```diff
@@ -6,10 +6,11 @@ class App extends React.Component {
   constructor(props) {
     super(props)
     this.state = {
-      stereos: []
+      stereos: [],
+      deepfryd_id: "37205-711",
     }
 
-    this.setState = this.setState.bind(this);
+    this.setState = this.setState.bind(thqis);
```

Comment:
You have a typo on line 13. (misspelled this)

Everything else looks good! 💃

Reply:
Nice catch, thanks!